### PR TITLE
Update release-notes for network-install BETA

### PR DESCRIPTION
**Pull request by timg236**
Include details of other changes plus instructions for disabling
network-install.

### CODE REVIEW

**Comment by timg236** [comment]
@peterharperuk

File: firmware/release-notes.md [hunk 1]
```diff
@@ -1,7 +1,20 @@
 # Raspberry Pi4 bootloader EEPROM release notes
 
 ## 2022-02-04 - Network Install - BETA
-   * New network install feature for the bootloader
+   * New network install feature for the bootloader. To disable network install
+     (e.g. in an industrial product) set NET_INSTALL_ENABLED=0 in the EEPROM
+     config or HDMI_DISABLE=1.
+     Network install is not enabled in secure-boot mode.
```
Comment (peterharperuk) [inline firmware/release-notes.md:7]: I don't think it's disabled for secure-boot at the moment

Comment (timg236) [reply firmware/release-notes.md:7]: Ah, good point. It just won't work :)

**Comment by peterharperuk** [comment]
looks good. I'll try and do a better job next time!

**Pull request merged**